scripts/core/EXT.rst

```diff
@@ -58,9 +78,9 @@ The following naming conventions must be followed for **experimental** extension
   - Experimental extensions are not guaranteed to be forward- or backward-compatible between versions.
   - Experimental extensions are not guaranteed to be supported in production driver releases; and may appear and disappear from release to release.
   - All extension functions must be postfixed with `Exp`
-  - All macros must use all caps `${X}_NAME_EXP` convention
-  - All structures, enumerations and other types must follow `${x}_name_exp_t` snake case convention
-  - All enumerator values must use all caps `${X}_ENUM_EXP_ETOR_NAME` convention
+  - All macros must use all caps with the appropriate prefix. Core macros shall use `${X}_NAME_EXP`, Sysman macros shall use `${S}_NAME_EXP`
+  - All structures, enumerations and other types must follow snake case convention with appropriate prefix. Core structures and enumerations shall use `${x}_name_exp_t` and Sysman shall use `${s}_name_exp_t`
+  - All enumerator values must use all caps with appropriate prefix. Core enumerator values shall use `${X}_ENUM_EXP_ETOR_NAME` and Sysman shall use `${S}_ENUM_EXP_ETOR_NAME`
```

Comment:
@wdamon-intel : should we add here convention for how the non-standard private extensions in the implementation should be named? like the one being adopted by the GPU driver here https://github.com/intel/compute-runtime/blob/10ac167bdc8a23d1e2dd5032171b391b1634c292/level_zero/include/zet_intel_gpu_metric.h#L43?

convention could be to have the vendor's name after the prefix, so:
macros:
ZE_<VENDOR>
ZES_<VENDOR>
ZETS_<VENDOR>

structs:
ze_<vendor>_
zes_<vendor>_
zet_<vendor>_

functions:
ze<Vendor>_
zes<Vendor>_
zet<Vendor>_

Reply:
Yes, I agree it would be an opportune time to introduce formally a non-standard private extension naming convention. Convention should be to have the vendor name after the prefix, yes.

Reply:
added this now, please check @wdamon-intel @jandres742